src/olympia/blocklist/utils.py

```diff
@@ -13,12 +24,19 @@ def block_activity_log_save(obj, change):
         'url': obj.url,
         'reason': obj.reason,
         'include_in_legacy': obj.include_in_legacy,
+        'comments': f'Versions {obj.min_version} - {obj.max_version} blocked.',
     }
-    log_create(
+    al = log_create(
         action, obj.addon, obj.guid, obj, details=details, user=obj.updated_by)
 
+    add_to_latest_version_in_channel(obj, al, amo.RELEASE_CHANNEL_LISTED)
+    add_to_latest_version_in_channel(obj, al, amo.RELEASE_CHANNEL_UNLISTED)
```

Comment:
mmm actually this is kind of weird. You're attaching logs to versions that might not have anything to do with the versions you're blocking, they might not even be in the same channel ? That seems wrong.

Reply:
Blocks aren't channel specific so they could affect either channel.  The issue did originally call for a log entry for each version and I asked if that was necessary, given the version is annotated with **Blocked** anyway, and Jorge seemed to think it wasn't.  

I could do it anyway though - it'd be iterating through every version and adding the `VersionLog` link if it matched.  
Some downsides here: 
- if old versions are blocked the message might not be visible in the log on the first page (we probably don't care)
- The logic around logging for editing blocks would be less clear.  E.g. if the range is shortened and a version now _isn't_ blocked what do we do about the existing message?  If the range is expanded do we have to add it to all the new affected versions?  If a block is edited generally do we add the message again to the existing versions?

Reply:
I think I'd rather have it set to all affected versions and only those, and re-add it when the block is edited - it's a log, let's treat it as append-only (plus it's an edge case anyway).

Reply:
https://github.com/mozilla/addons-server/pull/12960/commits/440add512e2381bf9a86430d876ed3eb520dcf95